.gitignore

```diff
@@ -1,6 +1,5 @@
 # Dependency directory
 node_modules
-build
```

Comment:
we can remove that its only for my fork because when i npm install my fork i will not have publish built script.